apps/web/src/components/PoolCandidatesTable/JobPlacementDialog.tsx

```diff
@@ -124,6 +126,12 @@ const JobPlacementDialog = ({
     RevertPlaceCandidate_Mutation,
   );
 
+  const { roleAssignments } = useAuthorization();
+  const canPlace = checkRole(
+    [ROLE_NAME.CommunityRecruiter, ROLE_NAME.CommunityAdmin],
```

Comment:
According to rolepermission.php, PoolOperators and RequestResponders can also change placement.

Reply:
Sure https://github.com/GCTC-NTGC/gc-digital-talent/pull/11273/commits/5ab10b60378595d9734fcdb1ea91a302710a9ab2